verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -163,6 +163,13 @@ def run_build(litani, jobs):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
+    # Restrict concurrency during CI to avoid 'out of memory' errors
+    # when proofs that require a lot of memory run concurrently
+    else:
+        factor   = 0.75
+        cpu_num  = os.cpu_count()
```

Comment:
This can return `None` on some platforms, you need to check for that

Reply:
Added a check for`cpu_num` being not `None` and greater than 1.